src/scripts/random-responses.js

```diff
@@ -94,8 +94,16 @@ const responseFrom =
     ) ?? [false, false, false, false];
 
     const message = { thread_ts: thread, unfurl_links: false };
+
     if (defaultEmoji) {
-      message.icon_emoji = defaultEmoji;
+      // The default emoji defined by the config may be a single emoji or a list.
+      // If it's a list, pick one at random.
+      if (Array.isArray(defaultEmoji)) {
+        message.icon_emoji =
+          defaultEmoji[Math.floor(Math.random() * defaultEmoji.length)];
```

Comment:
what happens if `defaultEmoji` is an array with length 0?

Reply:
![Trust me. It works.](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWZhYWJsdHkwZjdidnFva3p1YWQwcTBtdXI4OGRoa2tleHFoeTJsayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/Mab1lyzb70X0YiNLUj/giphy.gif)